src/Network/Network.Test/ScenarioTests/LoadBalancerTests.ps1

```diff
@@ -2554,6 +2554,213 @@ function Test-LoadBalancerCRUD-InternalStandardSku
+        Assert-AreEqual $tunnelInterface1.Identifier $expectedLb.BackendAddressPools[0].TunnelInterfaces[0].Identifier
+
+        Assert-AreEqual 1 @($expectedLb.BackendAddressPools[1].TunnelInterfaces).Count
+        Assert-AreEqual $tunnelInterface2.Protocol $expectedLb.BackendAddressPools[1].TunnelInterfaces[0].Protocol
+        Assert-AreEqual $tunnelInterface2.Type $expectedLb.BackendAddressPools[1].TunnelInterfaces[0].Type
+        Assert-AreEqual $tunnelInterface2.Port $expectedLb.BackendAddressPools[1].TunnelInterfaces[0].Port
+        Assert-AreEqual $tunnelInterface2.Identifier $expectedLb.BackendAddressPools[1].TunnelInterfaces[0].Identifier
+    }
+    finally
+    {
+        # Cleanup
+        Clean-ResourceGroup $rgname
+    }
+}
+
+<#
+.SYNOPSIS
+Tests Gateway LoadBalancer Consumer ref Provider.
+#>
+function Test-GatewayLoadBalancer-ConsumerLb
+{
+    # Setup Provider
+    $rgname = Get-ResourceGroupName
+    $vnetProviderName = Get-ResourceName
+    $subnetProviderName = Get-ResourceName
+    $lbProviderName = Get-ResourceName
+    $frontendProviderName = Get-ResourceName
+    
+    # Setup Provider
+    $pipConusmerName = Get-ResourceName
+    $subnetConsumerName = Get-ResourceName
+    $lbConsumerName = Get-ResourceName
+    $frontendConsumerName = Get-ResourceName
+    $domainNameLabel = Get-ResourceName
+
+    $rglocation = Get-ProviderLocation ResourceManagement
+    $resourceTypeParent = "Microsoft.Network/loadBalancers"
+    $location = Get-ProviderLocation $resourceTypeParent
+
+    try 
+    {
+        # Create resource group
+        $resourceGroup = New-AzResourceGroup -Name $rgname -Location $rglocation -Tags @{ testtag = "testval"} 
+        
+        # Create Provider Virtual Network
+        $subnet = New-AzVirtualNetworkSubnetConfig -Name $subnetProviderName -AddressPrefix 10.0.1.0/24
+        $vnet = New-AzVirtualNetwork -Name $vnetProviderName -ResourceGroupName $rgname -Location $location -AddressPrefix 10.0.0.0/16 -Subnet $subnet
+
+        # Create Provider LoadBalancer
+        $frontendProvider = New-AzLoadBalancerFrontendIpConfig -Name $frontendProviderName -Subnet $vnet.Subnets[0]
+        $lbProvider = New-AzLoadBalancer -Name $lbProviderName -ResourceGroupName $rgname -Location $location -FrontendIpConfiguration $frontendProvider -Sku Gateway
+
+        # Create Consumer publicip
+        $publicipConsumer = New-AzPublicIpAddress -ResourceGroupName $rgname -Name $pipConusmerName -location $location -AllocationMethod Static -DomainNameLabel $domainNameLabel -Sku Standard
+
+        # Create Consumer LoadBalancer
+        $lbConsumer = New-AzLoadBalancer -Name $lbConsumerName -ResourceGroupName $rgname -Location $location -Sku Standard
+        Add-AzLoadBalancerFrontendIpConfig -PublicIpAddress $publicipConsumer -GatewayLoadBalancerId $frontendProvider.Id -LoadBalancer $lbConsumer -Name $frontendConsumerName
```

Comment:
For Line 2746, parameter set cannot be resolved using the specified named parameters. One or more parameters issued cannot be used together or an insufficient number of parameters were provided.

Reply:
fixed